fluss-client/src/test/java/com/alibaba/fluss/client/table/FlussPartitionedTableITCase.java

```diff
@@ -58,6 +58,10 @@ void testPartitionedPrimaryKeyTable() throws Exception {
         Schema schema = createPartitionedTable(DATA1_TABLE_PATH_PK, true);
         Map<String, Long> partitionIdByNames =
                 FLUSS_CLUSTER_EXTENSION.waitUtilPartitionAllReady(DATA1_TABLE_PATH_PK);
+
+        TableDescriptor partitionTableDescriptor = TableDescriptor.builder().schema(schema).build();
+        createTable(TablePath.of("test_db_11", "tt"), partitionTableDescriptor, false);
```

Comment:
What's the purpose of this test code? It seems the created table is never used.

Reply:
Forget to remove these codelines for debug test failure...